composer.json

```diff
@@ -18,14 +18,16 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-gmp": "*",
```

Comment:
Argh, this is kinda heavy.

Reply:
Yeap, it reinforces my wish to split the lib in different small ones.